{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -168,3 +169,11 @@
 
 # This is needed to get a CRSF token in /admin
 ANON_ALWAYS = True
+
+USE_X_FORWARDED_HOST = config('USE_X_FORWARDED_HOST', default=True, cast=bool)
+SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default=not DEBUG, cast=bool)
+SECURE_HSTS_SECONDS = config('SECURE_HSTS_SECONDS', default='0', cast=int)
```

Comment:
I had some reservations settings this to something other than 0 by default to prevent accidental HSTS deployments but I hear you and you're right. After all we should be moving to a HTTPS only web.